src/api/useCartoProps.js

```diff
@@ -0,0 +1,18 @@
+import { DataFilterExtension } from '@deck.gl/extensions';
+import useViewport from './useViewport';
+import { applyFilter } from './Filter';
+import { debounce } from '../utils/debounce';
+
+export default function useCartoProps(source) {
```

Comment:
Didn't we talk about renaming this to be more explicit? like useCartoFilterProps?

Reply:
Recent commits have those changes!